From: Shengjing Zhu <zhsj@debian.org>
Date: Sun, 30 Aug 2020 18:45:23 +0800
Subject: Ignore all main_test

1. cut down golang-golang-x-tools-dev depends
2. test building and generated doc is useless in debian package
3. it wants to create config file in HOME

Forwarded: not-needed
---
 cmd/dlv/dlv_test.go | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/cmd/dlv/dlv_test.go b/cmd/dlv/dlv_test.go
index c4e903c..b0bbc51 100644
--- a/cmd/dlv/dlv_test.go
+++ b/cmd/dlv/dlv_test.go
@@ -1,3 +1,5 @@
+// +build ignore
+
 package main_test
 
 import (
